### ISSUE TITLE
zou 14.1: unable to delete task comment or change task status

### ISSUE THREAD

**Issue by mdk-man** [cgwire/zou#462]
On a fresh install a warning is displayed in Kitsu while deleting a task status.
The following error pops in gunicorn_errors.log:
```
[2022-05-06 16:04:21,996] ERROR in __init__: Traceback (most recent call last):
  File "/opt/zou/zouenv/lib/python3.8/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/opt/zou/zouenv/lib/python3.8/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/opt/zou/zouenv/lib/python3.8/site-packages/flask_restful/__init__.py", line 468, in wrapper
    resp = resource(*args, **kwargs)
  File "/opt/zou/zouenv/lib/python3.8/site-packages/flask/views.py", line 89, in view
    return self.dispatch_request(*args, **kwargs)
  File "/opt/zou/zouenv/lib/python3.8/site-packages/flask_restful/__init__.py", line 583, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/opt/zou/zouenv/lib/python3.8/site-packages/flask_jwt_extended/view_decorators.py", line 108, in wrapper
    return fn(*args, **kwargs)
  File "/opt/zou/zouenv/lib/python3.8/site-packages/zou/app/blueprints/tasks/resources.py", line 160, in delete
    tasks_service.reset_task_data(comment["object_id"])
  File "/opt/zou/zouenv/lib/python3.8/site-packages/zou/app/services/tasks_service.py", line 1391, in reset_task_data
    task_status_id = get_default_status().id
AttributeError: 'dict' object has no attribute 'id'
```

**Comment by frankrousseau** [comment]
It should be fixed in 0.14.2. Can you try again?

**Comment by mdk-man** [comment]
Perfect ! Thank you very much 😊

De : Frank Rousseau ***@***.***>
Envoyé : dimanche 8 mai 2022 16:59
À : cgwire/zou ***@***.***>
Cc : mdk-man ***@***.***>; Author ***@***.***>
Objet : Re: [cgwire/zou] zou 14.1: unable to delete task comment or change task status (Issue #462)


It should be fixed in 0.14.2. Can you try again?
